src/Protocol/Protocol_1_9.cpp

```diff
@@ -51,9 +51,11 @@ Implements the 1.9 protocol classes:
 
 
 
-/** Value for main hand in Hand parameter for Protocol 1.9. */
-static const UInt32 MAIN_HAND = 0;
-static const UInt32 OFF_HAND = 1;
+// Value for main hand in Hand parameter for Protocol 1.9.
+#define MAIN_HAND 0
+
+// Value for left hand in MainHand parameter for Protocol 1.9.
+#define LEFT_HAND 0
```

Comment:
Is this intended to be the same value?

Reply:
Yeah, the first is for main/offhand attack, second is for the left handed player/right handed player setting.